### PR TITLE
Validate graphicsmagick exists on Mac

### PR DESCRIPTION
**Pull request by nedSaf**
#4

### CODE REVIEW

**Comment by nedSaf** [comment]
test

**Comment by nedSaf** [comment]
@amitaibu Doesn't work with ubuntu.

File: generators/app/index.js [hunk 1]
```diff
@@ -108,6 +109,35 @@ module.exports = yeoman.generators.Base.extend({
       done();
     },
 
+    graphicsmagickExists: function () {
+      var done = this.async();
+
+      var self = this;
+
+      var graphicsmagickInstalled = false;
+      var graphicsmagickInfo = spawn('brew', ['info', 'graphicsmagick']);
```
Comment (amitaibu) [inline generators/app/index.js:123]: check `platform == darwin`

**Comment by nedSaf** [comment]
@amitaibu There's an error installing `composer` on Ubuntu:
selection_160

**Comment by amitaibu** [comment]
Check if you have composer installed globally.

**Comment by nedSaf** [comment]
Ok added it to `README`

File: generators/app/index.js [hunk 2]
```diff
@@ -108,6 +109,40 @@ module.exports = yeoman.generators.Base.extend({
       done();
     },
 
+    graphicsmagickExists: function () {
+      var done = this.async();
+
+      var self = this;
+
+      if (process.platform == 'darwin') {
```
Comment (amitaibu) [inline generators/app/index.js]:
``` js
if (process.platform != 'darwin') {
  return done();
}
```

Comment (nedSaf) [inline generators/app/index.js]: Nope. Doesn't work, `done()` doesn't stop it and return stops the whole process.

**Comment by amitaibu** [comment]
Ready for review?

**Comment by nedSaf** [comment]
Yes.

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request closed without merging**